MapboxNavigation/Style.swift

```diff
@@ -147,11 +147,27 @@ open class DistanceLabel: StylableLabel { }
 /// :nodoc:
 @objc(MBDestinationLabel)
 open class DestinationLabel: StylableLabel {
-    typealias AvailableBoundsHandler = () -> (CGRect)
-    var availableBounds: AvailableBoundsHandler!
+//    typealias AvailableBoundsHandler = () -> (CGRect)
+//    var availableBounds: AvailableBoundsHandler!
+//    var unabridgedText: String? {
+//        didSet {
+//            super.text = unabridgedText?.abbreviated(toFit: availableBounds(), font: font)
+//        }
+//    }
+    
     var unabridgedText: String? {
         didSet {
-            super.text = unabridgedText?.abbreviated(toFit: availableBounds(), font: font)
+            //super.text = unabridgedText?.abbreviated(toFit: availableBounds(), font: font)
+            super.text = unabridgedText
+            sizeToFit()
+        }
+    }
+    
+    var smartText: String? {
```

Comment:
Is this used?

Reply:
Not yet but the intention here was to support something like `[Shield][String][Image]` depending on what the format will look like in MapboxDirections.swift. Parsing, async loading of images/shields, abbreviations, layout and constructing an attributed string will be done by setting smartText or whatever we call it later.